solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -120,7 +117,7 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
 
   // Add file to dataset
   router.get(
-    '/datasets/:dataSetId/upload-requests',
+    '/projects/:projectsId/datasets/:dataSetId/upload-requests',
```

Comment:
again, there's a risk here depending on how authz is generated (and how it is updated in the future).

If authz gets set up so it happens against the projectId (or updated to that in the future), one could give a projectId to which they have access, but a complete different dataset Id and potentially pollute the dataset with unwanted (or malicious) files.